devfile.yaml

```diff
@@ -19,15 +20,16 @@ starterProjects:
     zip:
       location: https://code.quarkus.redhat.com/d?e=io.quarkus%3Aquarkus-resteasy&e=io.quarkus%3Aquarkus-smallrye-health&e=io.quarkus%3Aquarkus-openshift
 components:
-  - name: buildguidance
-    attributes:
-      tool: console-import
-    container:
-      image: buildguidanceimage-placeholder
-      memoryLimit: 1024Mi
-      endpoints:
-        - name: http-8081
-          targetPort: 8081
+  - name: outerloop-build
+    image:
+      imageName: java-quarkus-image:latest
+      dockerfile:
+        uri: src/main/docker/Dockerfile.jvm.staged
+        buildContext: .
+        rootRequired: false
+  - name: outerloop-deploy
+    kubernetes:
+      uri: outerloop-deploy.yaml
   - name: tools
     container:
       image: quay.io/eclipse/che-quarkus:nightly
```

Comment:
Can we update tools container of those devfiles to use the developer image like `quay.io/devfile/universal-developer-image:ubi8-d433ed6` instead of the `quay.io/eclipse/che-quarkus:nightly` images that no longer exists ?

Reply:
https://github.com/devfile-samples/devfile-sample-java-springboot-basic/pull/1#discussion_r746960486